### DISCUSSION TITLE
Latex support in comment markdown (Android)

### DISCUSSION THREAD

**Discussion by aeryskyB** [community/community#54076] [Mobile]
I am not sure if this is only happening to me. Whenever I look into the issues, I find the raw latex command, not the result.
___
Best!

**Comment by hrishikesh-kocharekar-dev** [comment]
It sounds like the LaTeX code is not being rendered properly in the GitHub issues. By default, GitHub doesn't render LaTeX code in its Markdown parser, so you may need to use a third-party service or extension to render the LaTeX code properly.

One option is to use a browser extension like TeX All the Things or GitHub with MathJax, which can render LaTeX code in the GitHub interface. Alternatively, you can use an external service like QuickLaTeX or KaTeX to render the LaTeX code and then paste the resulting image or HTML into the issue.

You can also consider using GitHub Flavored Markdown (GFM) to render LaTeX code within your issues. GFM supports a limited subset of LaTeX commands, which can be enclosed in `$` symbols or `$$` symbols for inline and display math, respectively. For example:

The quadratic formula is given by $x = \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}$.

This will render as:

The quadratic formula is given by $x = \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}$.

Note that this method has limitations and may not work for more complex LaTeX expressions.

**Reply by aeryskyB** [reply]
It sure renders on web browsers. But, the Android app fails to do so.

**Reply by VerasNp** [reply]
Same issue here in my notes I wrote some LaTeX mixed with Markdown... in the web browser it renders the LaTeX but in the app it doesn't.

**Comment by markcst** [comment]
It's almost 2025 and Microsoft still hasn't done this crap?
And move along, you just need to implement Latex rendering in an app, not go to the moon!
